src/common/motech-list/list.controller.js

```diff
@@ -44,6 +47,7 @@
 
         var readyTimeout;
         function broadcastReady(){
+            $scope.isLoading = false;
```

Comment:
I'm not sure what you are trying to accomplish here. The broadcastReady is an internal display function -- if you want to unset loading when first loaded and ready, it might make most sense to subscribe to the 'motech-list.ready'

It might make sense to stop broadcastReady from being called is "isLoading" is true